data/mods/PKs_Rebalance/pk_terrain.json

```diff
@@ -0,0 +1,108 @@
+[
+    {
+        "type" : "terrain",
+        "id" : "t_guardrail_bg_dp",
+        "edit-mode": "modify",
+        "name": "guard rail",
+        "symbol": "#",
+        "color": "ltgray",
+        "move_cost": 3,
+        "flags": ["TRANSPARENT", "NOITEM", "REDUCE_SCENT", "MOUNTABLE", "SHORT", "THINOBSTACLE", "ROAD"],
```

Comment:
I want to try and handle conflicting flags and the like. If you want to help extend the ruleset help would be appreciated.